pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/VarByteChunkForwardIndexWriterV4.java

```diff
@@ -121,6 +123,42 @@ public void putBigDecimal(BigDecimal bigDecimal) {
     putBytes(BigDecimalUtils.serialize(bigDecimal));
   }
 
+  @Override
```

Comment:
(minor) Suggest keeping the method the same order as in the interface to be easier to navigate